### DISCUSSION TITLE
We've yet again been nominated for a Webby Award!

### DISCUSSION THREAD

**Discussion by DustinBrett** [DustinBrett/daedalOS#421] [Announcements]
**Big news!** 🎉 My personal website has once again been nominated for [The Webby Awards](https://www.webbyawards.com/)! Please take a moment to [vote for me](https://vote.webbyawards.com/PublicVoting#/2024/websites-and-mobile-sites/general-desktop-mobile-sites/personal-blogwebsite) to win the People's Voice Award!

**Comment by bhardvaja** [comment]
awesome

**Comment by DustinBrett** [comment]
Whelp tomorrow is the last day to vote and daedalOS is in 4th place. I thank everyone who voted and I think my only hope now is to win the judges vote. If I am being honest the judges vote is the one I more desire as I didn't expect to win a popularity contest like the People's Voice when I don't have much of a social media presence. I have my fingers crossed that the judges notice my hard work, and if it doesn't work out, I will rethink my strategy for next year and also work on making my site more accessible to a wider audience.
